server/src/workspace/workspace-manager/workspace-manager.service.ts

```diff
@@ -255,7 +255,6 @@ export class WorkspaceManagerService {
    */
   public async delete(workspaceId: string): Promise<void> {
     // Delete data from metadata tables
-    await this.fieldMetadataService.deleteFieldsMetadata(workspaceId);
```

Comment:
There was a bug when we were deleting fields before deleting relations.
Now that cascade deletions are fixed on fields and relations, deleting objectMetadata should delete all fields and relations and we can delegate that part to postgres.